sdg/components.py

```diff
@@ -22,22 +22,22 @@ def git_clone_op(
             # Increase logging verbosity
             set -x &&
 
-            # Add TLS Parameters if CA Cert exists and is non-zero size
-            ADDITIONAL_CLONE_PARAMS=""
+            # Set Preferred CA Cert
             if [ -s "$TAXONOMY_CA_CERT_PATH" ]; then
-                ADDITIONAL_CLONE_PARAMS="-c http.sslVerify=true -c http.sslCAInfo=$TAXONOMY_CA_CERT_PATH"
+                export GIT_SSL_NO_VERIFY=false
```

Comment:
Is this not the default value? It seems like it's unnecessary to set this. The comment applies for all exports of this environment variable.

Reply:
I `true` believe it is the default, but felt it would be safer to explicitly set this to guarantee cert validation is used